bliss/sle/cltu.py

```diff
@@ -148,8 +148,8 @@ def start(self):
         '''
         start_invoc = CltuUserToProviderPdu()
 
-        if self._credentials:
-            pass
+        if self._auth_level in ['all']:
```

Comment:
Can we set this to be `_auth_level == 'all'` instead of doing membership checks?